### ISSUE TITLE
VISITOR panel wrong + Unknown Browser and OS

### ISSUE THREAD

**Issue by rusk0** [allinurl/goaccess#2517]
Greeting, i was hoping i could get a clue what am i doing wrong with the following example :

Version:

**GoAccess - 1.7.2**

Example log:

`10.x.x.x - - - some.domain.com [31/May/2023:09:43:04 +0000] "GET //some/url/path?_=some-query-string HTTP/1.1" - upstream_response_time - msec 1685526184.051 request_time 0.000 304 0 "-" Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36 Edg/112.0.1722.48 1.2.3.4 user:'-'origin:'https://www.some-origin.com'`

Command i run:

`/usr/bin/goaccess /var/log/nginx/access.log -o /var/www/html/index.html --log-format='%h %^ %^ %^ %v [%d:%t %^] %~ %m %U %H %^ %^ %T %^ %^ %^ %^ %s %b %R %u %~h %^' --real-time-html --daemonize`

The result is that Visitors panel has this data :
31/May/2023
And Browser and OS fields have 'Unknown' data.

Everything else works. 
Additionally i was hoping i can swap the %h field in the VISITOR HOSTNAMES AND IPS panel  with the X-Forwarded-For value, taken with %~h , so i though of changing the above log format to :
`'%^ %^ %^ %^ %v [%d:%t %^] %~ %m %U %H %^ %^ %T %^ %^ %^ %^ %s %b %R %u %h %^'`

But if i do i get :
 Token 'illa/5.0' doesn't match specifier '%h'
 
 It seems to me like the %u placeholder doesn't capture the entire user-agent string ? 
 I am however totally confused where the 31/May/2023 info come in VISITORS panel ?

**Comment by allinurl** [comment]
The current example does not specify where the user agent ends, so you would need a way to delimit it. If you add quotes around, this should do it:

    goaccess access.log --log-format='%h %^ %^ %^ %v [%d:%t %^] "%r" %^ %^ %^ %^ %^ %^ %T %s %^ "%R" "%u" %^' --date-format=%d/%b/%Y --time-format=%T

**Comment by rusk0** [comment]
Thank you for you input. 

The missing quotes were indeed the issue - sorry for missing that and thanks for pointing this out. 
And obviously i wasn't getting the VISITOR panel idea, but now i do - so that's fine as well.
I also managed to work out the X-Forwarded-For header to be displayed instead of the client IP . 

All issues with my custom log format were resolved. 

Awsome tool !!!

Thanks once more for your input !
